test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/ObjectsTest.scala

```diff
@@ -94,6 +94,30 @@ class ObjectsTest {
     assertEquals("abc", ju.Objects.requireNonNull("abc", ""))
   }
 
+  @Test def requireNonNullWithMsgSupplier(): Unit = {
+    val message =  "All is well!"
+
+    val successSupplier = new ju.function.Supplier[String] {
+          def get(): String = message
+      }
+
+    val failureSupplier = new ju.function.Supplier[String] {
+        def get(): String = throw new AssertionError(
+            "Objects.requireNonNull() should not have called Supplier")
+      }
```

Comment:
A bunch of whitespace fixes:
```suggestion
    val message = "All is well!"

    val successSupplier = new ju.function.Supplier[String] {
      def get(): String = message
    }

    val failureSupplier = new ju.function.Supplier[String] {
      def get(): String = {
        throw new AssertionError(
            "Objects.requireNonNull() should not have called Supplier")
      }
    }
```